build/yamls/flow-aggregator/base/flow-aggregator.yml

```diff
@@ -24,7 +24,7 @@ rules:
     verbs: ["get", "list", "watch"]
   - apiGroups: [""]
     resources: ["configmaps"]
-    verbs: ["create", "get", "list", "watch"]
+    verbs: ["create", "get", "list", "watch", "update", "patch"]
```

Comment:
can we restrict permissions appropriately? I think that only the `flow-aggregator-configmap-xxx` ConfigMap should be included here. At the moment you are giving the FlowAggregator permissions to mutate all ConfigMaps in all Namespaces...

Reply:
Thanks for reminding. I've changed it.